modules/siddhi-launcher/src/main/java/org/wso2/siddhi/sdk/launcher/debug/VMDebugSession.java

```diff
@@ -0,0 +1,169 @@
+/*
+*  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing,
+*  software distributed under the License is distributed on an
+*  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+*  KIND, either express or implied.  See the License for the
+*  specific language governing permissions and limitations
+*  under the License.
+*/
+
+
+package org.wso2.siddhi.sdk.launcher.debug;
+
+import io.netty.channel.Channel;
+import org.wso2.siddhi.core.debugger.SiddhiDebugger;
+import org.wso2.siddhi.sdk.launcher.debug.dto.BreakPointDTO;
+import org.wso2.siddhi.sdk.launcher.debug.internal.DebugRuntime;
+import org.wso2.siddhi.sdk.launcher.exception.DebugException;
+
+import java.util.List;
+
+/**
+ * {@code VMDebugSession} The Debug Session class will be used to hold context for each client.
+ * Each client will get its own instance of debug session.
+ */
+public class VMDebugSession {
+
+    private Channel channel = null;
+
+    private DebugRuntime debugRuntime = null;
+
+    public VMDebugSession() {
```

Comment:
You do not need to add default constructor. java will add this automatically

Reply:
Resolved in commit dda44a8.